docs/source/install/installation.rst

```diff
@@ -197,17 +197,25 @@ Configure AiiDA
 
 Verdi tab-completion
 --------------------
-The ``verdi`` command line interface has many commands and options.
-To simplify its usage, there is a way to enable tab-completion for it in your shell.
-To do so, simply add the following line to the activation script of your virtual environment (or to your shell config, e.g. ``.bashrc``)::
+The ``verdi`` command line interface has many commands and options,
+which can be tab-completed to simplify your life.
+Enable tab-completion with the following bash command::
```

Comment:
This will actual work for various shells, `click` will attempt to detect the shell and format the correct autocomplete string. This works at least for bash and zsh

Reply:
Ok, I'll replace "bash" by "shell"